p2p/client.go

```diff
@@ -230,14 +230,14 @@ func (c *Client) RemoveBlocks(ctx context.Context, from, to uint64) error {
 	}
 
 	for h := from; h < to; h++ {
-
 		cid, err := c.store.LoadBlockCid(h)
 		if err != nil {
-			return fmt.Errorf("load block id from store %d: %w", h, err)
+			c.logger.Error("load blocksync block id from store", "height", h, "error", err)
+			continue
```

Comment:
>  there is no cid stored for a specific height, which may happen

why may it happen?

Reply:
I guess it can happen a lot right, since blocks maybe came from another source 👍

Reply:
do you even need to log error then? probably want to differentiate not found vs parse error

Reply:
done